duckduckgo/iOS
# Update tab switcher to scroll vertically

Reviewer: Caine
Asana: https://app.asana.com/0/19626896268257/349456047677815

**Description**:
Updated the tab switcher to scroll vertically. Hopefully another easy one to review!

**Steps to test this PR**:
• run the app
• open enough tabs to fill the screen
• note that the tab switcher now scrolls vertically (it previously scrolled horizontally)


###### Reviewer Checklist:


###### PR DRI Checklist:
